README.md

```diff
@@ -117,89 +117,3 @@ TODOs:
 
 1.  hook up a lit test target.
 2.  re-add npcomp instructions once it is upgraded to use the same build setup.
-
```

Comment:
Are the related files removed completely? If not, may be move this to a README.md in the directory with those files.

Reply:
Yes they are removed as per Deny's suggestion (internally).